src/vmm_unix.ml

```diff
@@ -488,3 +488,42 @@ let find_block_devices () =
         | Ok size, Ok id ->
           Ok ((id, size) :: acc))
     (Ok []) files
+
+external cpu_count : unit -> int = "vmm_cpu_count"
+
+external disk_space : string -> int = "vmm_disk_space"
+
+external memory : unit -> int = "vmm_memory"
+
+let find_bridges () =
+  match Lazy.force uname with
+  | FreeBSD ->
+    let cmd = Bos.Cmd.(v "ifconfig" % "-g" % "bridge") in
+    let* names = Bos.OS.Cmd.(run_out cmd |> out_lines |> success) in
+    Ok names
+  | Linux ->
+    let* bridges = Bos.(OS.Cmd.(run_out Cmd.(v "ip" % "-o" % "link" % "show" % "type" % "bridge") |> out_lines |> success)) in
+    (* output is <id>: <name>: ... *)
+    Ok (List.fold_left (fun acc s ->
+        match String.split_on_char ':' s with
+        | _id :: name :: _tl -> String.trim name :: acc
+        | _ -> Logs.err (fun m -> m "couldn't find bridge name in %s" s); acc)
+        [] bridges)
+
+let root_policy () =
+  try
+    let cpus = cpu_count () in
+    let disk_space = disk_space (Fpath.to_string (block_dir ())) in
+    let memory = memory () in
+    let* bridges = find_bridges () in
+    let rec gen_cpu acc n =
+      if n = 0 then acc else gen_cpu (Vmm_core.IS.add (pred n) acc) (pred n)
+    in
+    Ok { Vmm_core.Policy.vms = max_int ;
+         cpuids = gen_cpu Vmm_core.IS.empty cpus ;
```

Comment:
A potential helper function `vm_count_of_cpu` could be a list of tuples that indicate how many vms are currently bound to each `cpu`. Will this be better on albatross or mollymawk?

Something like:
```ocaml
[(1,20);(2,0)] 
``` 
etc

Reply:
I'm not keen on modifying the policy type. I think in mollymawk there could be some of that information, i.e.:
- we have the root policy, seeing that CPUs 0..19 exist
- we have two policies "foo" with cpu id 0,1,2,3,4,5 and "bar" with cpu id 6, 7, 8, 9, 10
- we display the information in a way that 0..10 are "already in use" (but can as well be assigned), and 11..19 are not used at all

Reply:
so, indeed this fits better into mollymawk from my point of view.

Reply:
Indeed I think this approach is much better